crane/template.go

```diff
@@ -0,0 +1,131 @@
+package crane
+
+import (
+	"fmt"
+	"github.com/michaelsauter/crane/print"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+type TemplateInfo struct {
```

Comment:
Maybe I am too defensive and internal interfaces shouldn't be public as they are now, but I feel all exposed structures should be subset interfaces of the internal ones - so it's clear that we break the external contract if we remove/update an internal function/struct/whatever.